### PR TITLE
34 feedback component

### PR DESCRIPTION
**Pull request by MohammadAlhalaq**
Create feedback component with style

### CODE REVIEW

File: client/src/components/feedback/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            <div className="table-number">
+              <p>5</p>
+              <h4>Table</h4>
+            </div>
+          </div>
+          <div>{errorMessage}</div>
+          <form>
+            <input
+              type="text"
+              onChange={this.setEmail}
```
Comment (ahmedisam99) [inline client/src/components/feedback/index.js]: Use the full definition of `controlled components` with your inputs, which means your state **must** always be your only source of truth.
https://reactjs.org/docs/forms.html#controlled-components

File: client/src/components/feedback/style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+/*
```
Comment (ahmedisam99) [inline client/src/components/feedback/style.css]: Unnecessary commented code.
remove this commented code.

File: client/src/components/feedback/style.css [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  line-height: 1.4rem;
+  padding: .45rem;
+}
+.order-number{
+  width: 40vw
+}
+.table-number{
+  width:40vw
+}
+form{
```
Comment (ahmedisam99) [inline client/src/components/feedback/style.css]: **Do not** use tag name selectors in css!!

File: package.json [hunk 4]
```diff
@@ -36,10 +36,13 @@
   "dependencies": {
     "@hapi/joi": "^16.1.4",
     "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
```
Comment (ahmedisam99) [inline package.json]: After using `proxy` in react there is no need for this module/package
```suggestion
   
```

File: package.json [hunk 5]
```diff
@@ -36,10 +36,13 @@
...
     "env2": "^2.2.2",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "pg": "^7.12.1"
+    "pg": "^7.12.1",
+    "react-router-dom": "^5.1.1",
```
Comment (ahmedisam99) [inline package.json]: Why is `react-router-dom` and `react-stars` installed in the backend?!?!?!?!!?!??!?!?!
It's obviously a mistake and not intended, but you must fix it before pushing it!!

File: server/app.js [hunk 6]
```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
+const cros = require('cors');
```
Comment (ahmedisam99) [inline server/app.js]: no need for cors anymore as I mentioned in a previous comment

File: client/src/components/feedback/index.js [hunk 7]
```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+import ReactStars from 'react-stars';
```
Comment (ahmedisam99) [inline client/src/components/feedback/index.js]: Rearrange your imports to look appropriate, see what your team mates did

File: client/src/components/feedback/index.js [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  };
+
+  setFeedback = event => this.setState({ content: event.target.value });
+
+  setEmail = event => this.setState({ email: event.target.value });
+
+  newRate = newR => this.setState({ rate: newR });
+
+  sendFeedback = () => {
+    const { history, orderId = 1 } = this.props;
```
[the remark this answers is not in the document]
Comment (MohammadAlhalaq) [reply client/src/components/feedback/index.js]: We will receive the order id as component from order as props  isnt that ?

[the remark this answers is not in the document]
Comment (MohammadAlhalaq) [reply client/src/components/feedback/index.js]: The order id will get from order component from fetch and will pass Id as props isn't that !?

**Pull request merged**